### ISSUE TITLE
we should emphasize reusability/reuse for recently added slots 

### ISSUE THREAD

**Issue by turbomam** [microbiomedata/nmdc-schema#924]
If we had a class called `PizzaCooking`, then we wouldn't need a `pizza_cooking_temp` slot. We would reuse a `temp` slot already existing in the schema if possible. If we had to create a new slot, we would still call it `temp` if possible!

@aclum I'm tagging you because this applies to soem of the library preparation and nucleic acid extraction modeling, but you are definitely not the only person who has overlooked this principle.

**Comment by turbomam** [comment]
`extraction_date` and `processed_date` for example

qualifying the data is only required if there are two or more date slots in a class

**Comment by turbomam** [comment]
see also:
- #917

**Comment by aclum** [comment]
I agree, we could used processed_date. The main thing Im worried about for date is some processes last more than one day so we may to use something like start_date and stop_date. This is more complicated than a single date but is more futureproof. [WorkflowExecutionActivity](https://microbiomedata.github.io/nmdc-schema/WorkflowExecutionActivity/) has something similar but with time included, [started_at_time](https://microbiomedata.github.io/nmdc-schema/started_at_time/) and [ended_at_time](https://microbiomedata.github.io/nmdc-schema/ended_at_time/). @mslarae13 @anastasiyaprymolenna  do you have input on if date is sufficient or if we need date time for `PlannedProcess`
